lemur/certificates/views.py

```diff
@@ -1650,6 +1650,78 @@ def put(self, certificate_id, data=None):
             return dict(message=f"Failed to revoke: {str(e)}"), 400
 
 
+class CertificateDeactivate(AuthenticatedResource):
+    def __init__(self):
+        self.reqparse = reqparse.RequestParser()
+        super(CertificateDeactivate, self).__init__()
+
+    def put(self, certificate_id):
+        """
+        .. http:put:: /certificates/1/deactivate
+
+           deactivate a certificate (integration test only)
+           **Example request**:
+
+           .. sourcecode:: http
+
+              PUT /certificates/1/deactivate HTTP/1.1
+              Host: example.com
+              Accept: application/json, text/javascript
+              Content-Type: application/json;charset=UTF-8
+
+           **Example response**:
+
+           .. sourcecode:: http
+
+              HTTP/1.1 200 OK
+              Vary: Accept
+              Content-Type: text/javascript
+
+              {
+                "id": 1
+              }
+
+           :reqheader Authorization: OAuth token to authenticate
+           :statuscode 200: no error
+           :statuscode 403: unauthenticated or cert attached to LB
+           :statuscode 400: encountered error, more details in error message
+
+        """
+        cert = service.get(certificate_id)
+
+        if not cert:
+            return dict(message="Cannot find specified certificate"), 404
```

Comment:
I would recommend using `412 = Precondition Failed` instead of 404, as it gets confusing whether the endpoint url is wrong giving 404 or an actual error thrown by API is giving 404

Reply:
shall we do this consistently across all endpoints checking for the certificate?